OpenSim/Actuators/VandenBogert2011Muscle.h

```diff
@@ -0,0 +1,568 @@
+
+        //OpenSim_DECLARE_PROPERTY(max_isometric_force, double,
+        //  F_{max} (N) "Maximum isometric force that the fibers can generate");
+        // Comes from parent class, Muscle.h
+
+        OpenSim_DECLARE_PROPERTY(fMaxTendonStrain, double,
+        "tendon strain at maximum isometric muscle force");
+        //u_{max} (dimensionless) strain in the series elastic element at load
+        // of maxIsometricForce
+
+        OpenSim_DECLARE_PROPERTY(fl_width, double,
+        "force-length shape factor");
+        //W (dimensionless) width parameter of the force-length relationship of
+        // the muscle fiber
+
+        OpenSim_DECLARE_PROPERTY(fv_AHill, double,
+        "force-velocity shape factor");
+        //AHill (dimensionless) Hill parameter of the force-velocity
+        // relationship
+
+        OpenSim_DECLARE_PROPERTY(fv_maxMultiplier, double,
+        "maximum normalized lengthening force");
+        //FV_{max} (dimensionless) maximal eccentric force
+
+        //OpenSim_DECLARE_PROPERTY(optimal_fiber_length, double,
+        //    "Optimal Length of Contractile Element"
+        //Lceopt (m)
+        // Comes from parent class, Muscle.h
+
+        OpenSim_DECLARE_PROPERTY(dampingCoefficient, double,
+        "The linear damping of the fiber");
+        //b (s/m) damping coefficient of damper parallel to the fiber
+        //      (normalized to maxIsometricForce)
+
+        OpenSim_DECLARE_PROPERTY(normFiberSlackLength, double,
+                                 "(dimensionless) slack length of the parallel "
+                                         "elastic element, divided by Lceopt");
+        //L_{slack,fiber}(dimensionless) slack length of the fiber (PEE)
+
+
+        OpenSim_DECLARE_PROPERTY(activTimeConstant, double,
+        "Activation time(s)");
+        //T_{act} (s) Activation time
+
+        OpenSim_DECLARE_PROPERTY(deactivTimeConstant, double,
+        "Deactivation time(s)");
+        //T_{deact} (s) Deactivation time
+
+        OpenSim_DECLARE_PROPERTY(pennAtOptFiberLength, double,
+                                 "pennation at optimal fiber length "
+                                         "equilibrium");
+
+        OpenSim_DECLARE_PROPERTY(defaultActivation, double,
+                                 "default activation");
+
+        OpenSim_DECLARE_PROPERTY(default_fiber_length, double,
+                                 "Assumed initial fiber length if none is "
+                                         "assigned.");
+
+
+//=============================================================================
+// Construction
+//=============================================================================
+        /**Default constructor: produces a non-functional empty muscle*/
+        VandenBogert2011Muscle();
+
+        /** Constructs a functional muscle using default parameters.
+         @param name The name of the muscle.*/
+        VandenBogert2011Muscle(const std::string& name);
```

Comment:
This constructor should be marked `explicit VandenBogert2011Muscle()`... But I prefer to just remove this constructor. It is not very common for OpenSim classes to have a conscructor that takes a name; they usually take an XML file name and so it can be confusing if some classes take a std::string for a name and others for a file name.